pkg/cmd/adm/install_operator.go

```diff
@@ -0,0 +1,239 @@
+	clicontext "github.com/kubesaw/ksctl/pkg/context"
+	"github.com/kubesaw/ksctl/pkg/ioutils"
+	olmv1 "github.com/operator-framework/api/pkg/operators/v1"
+	olmv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	errs "github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/json"
+	"k8s.io/apimachinery/pkg/util/wait"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/spf13/cobra"
+)
+
+type installArgs struct {
+	kubeConfig string
+	namespace  string
+}
+
+func NewInstallOperatorCmd() *cobra.Command {
+	commandArgs := installArgs{}
+	cmd := &cobra.Command{
+		Use:   "install-operator <host|member> --kubeconfig <path/to/kubeconfig> --namespace <namespace>",
+		Short: "install kubesaw operator (host|member)",
+		Long:  `This command installs the latest stable versions of the kubesaw operator using OLM`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			term := ioutils.NewTerminal(cmd.InOrStdin, cmd.OutOrStdout)
+			kubeClient, err := client.NewKubeClientFromKubeConfig(commandArgs.kubeConfig)
+			if err != nil {
+				return err
+			}
+			ctx := clicontext.NewTerminalContext(term, kubeClient)
+			return installOperator(ctx, commandArgs, args[0], time.Second*60)
+		},
+	}
+
+	cmd.Flags().StringVar(&commandArgs.kubeConfig, "kubeconfig", "", "Path to the kubeconfig file to use.")
+	flags.MustMarkRequired(cmd, "kubeconfig")
+	cmd.Flags().StringVar(&commandArgs.namespace, "namespace", "", "The namespace where the operator will be installed. Host and Member should be installed in separate namespaces. If the namespace is not provided the standard namespace names are used: toolchain-host|member-operator.")
+	return cmd
+}
+
+func installOperator(ctx *clicontext.TerminalContext, args installArgs, operator string, timeout time.Duration) error {
+	// validate cluster type
+	if operator != string(configuration.Host) && operator != string(configuration.Member) {
+		return fmt.Errorf("invalid operator type provided: %s. Valid ones are %s|%s", operator, string(configuration.Host), string(configuration.Member))
+	}
+
+	// assume "standard" namespace if not provided
+	namespace := args.namespace
+	if args.namespace == "" {
+		namespace = fmt.Sprintf("toolchain-%s-operator", operator)
+	}
+
+	if !ctx.AskForConfirmation(
+		ioutils.WithMessagef("install %s in namespace '%s'", operator, namespace)) {
+		return nil
+	}
+
+	// check that we don't install both host and member in the same namespace
+	if err := checkOneOperatorPerNamespace(ctx, namespace, operator); err != nil {
+		return err
+	}
+
+	// install the catalog source
+	catalogSourceKey := types.NamespacedName{Name: operatorResourceName(operator), Namespace: namespace}
+	catalogSource := newCatalogSource(catalogSourceKey, operator)
+	if err := ctx.KubeClient.Create(ctx, catalogSource); err != nil {
+		return err
+	}
```

Comment:
Let's add a few more logs for each creation:
```suggestion
	ctx.Println(fmt.Sprintf("Creating CatalogSource %s in namespace %s.", catalogSource.Name, catalogSource.Namespace))
	if err := ctx.KubeClient.Create(ctx, catalogSource); err != nil {
		return err
	}
	ctx.Println(fmt.Sprintf("CatalogSource %s created.", catalogSource.Name))
```